frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedPanel.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {
+  ResourceIcon,
+  ResourceLink,
+  SimpleTabNav,
+  ResourceSummary,
+  SectionHeading,
+} from '@console/internal/components/utils';
+import { referenceFor } from '@console/internal/module/k8s';
+import TopologyOperatorBackedResources from './TopologyOperatorBackedResources';
+import { TopologyDataObject } from '../topology-types';
+
+export type TopologyOperatorBackedPanelProps = {
+  item: TopologyDataObject<string>;
+};
+
+const TopologyOperatorBackedPanel: React.FC<TopologyOperatorBackedPanelProps> = ({ item }) => {
+  const {
+    name,
+    resources: { obj },
+  } = item;
+
+  const ResourcesSection = () => <TopologyOperatorBackedResources item={item} />;
+  const DetailsSection = () => (
+    <div className="co-m-pane__body">
```

Comment:
This will align the resource details better with heading and nav items of the sidebar. There's a difference of `10px` between sidebar heading and resource details.
```suggestion
    <div className="overview__sidebar-pane-body">
```

Reply:
Updated.